scripts/pegasus_online_node_list.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
```

Comment:
There are many duplicate code with `pegasus_offline_node_list.sh`, could you please combine the two scripts into a single one?